extra/docs_landing.html

```diff
@@ -1,238 +1,366 @@
+        </dd>
+      </dl>
+      
+      <h3 id="_store">Store</h3>
+      <dl>
+        <dt id="stack-components-elasticsearch">
+          Elasticsearch
+        </dt>
+        <dd>
+          Elasticsearch is the distributed search and analytics engine at the heart of the Elastic Stack. It provides near real-time search and analytics for all types of data. Whether you have structured or unstructured text, numerical data, or geospatial data, Elasticsearch can efficiently store and index it in a way that supports fast searches. Elasticsearch provides a REST API that enables you to store data in Elasticsearch and retrieve it. The REST API also provides access to Elasticsearch&#8217;s search and analytics capabilities. <a href="en/elasticsearch/reference/current/elasticsearch-intro.html">Learn more about Elasticsearch</a>.
+        </dd>
+      </dl>
+      
+      <h3 id="_consume">Consume</h3>
+      <p>Use Kibana to query and visualize the data that&#8217;s stored in Elasticsearch. Or, use the Elasticsearch clients to access data in Elasticsearch directly from common programming languages.</p>
+      <dl>
+        <dt id="stack-components-kibana">
+          Kibana
+        </dt>
+        <dd>
+          Kibana is the tool to harness your Elasticsearch data and to manage the Elastic Stack. Use it to analyze and visualize the data that&#8217;s stored in Elasticsearch. Kibana is also the home for the Elastic Enterprise Search, Elastic Observability and Elastic Security solutions. <a href="en/kibana/current/introduction.html">Learn more about Kibana</a>.
+        </dd>
+        <dt id="stack-components-elasticsearch-clients">
+          Elasticsearch clients
+        </dt>
+        <dd>
+          The clients provide a convenient mechanism to manage API requests and responses to and from Elasticsearch from popular languages such as Java, Ruby, Go, Python, and others. Both official and community contributed clients are available. <a href="en/elasticsearch/client/index.html">Learn more about the Elasticsearch clients</a>.
+        </dd>
+      </dl>
+    </details>
+    <p>
+      <a class="inline-block mr-3" href="https://www.elastic.co/guide/index.html#viewall">All Elastic docs</a> 
+      <a class="inline-block mr-3" href="https://www.elastic.co/guide/en/welcome-to-elastic/current/new.html">Release docs</a>
+      <a class="inline-block mr-3" href="https://www.elastic.co/videos">Videos & Webinars</a>
+   </p>
+   </div>
+   <div class="col-md-6 col-12">
+     <iframe class="vidyard_iframe" src="https://play.vidyard.com/qAS1W24928bBrc2sv5f5Ti?v=4.3&amp; width="640" height="360" scrolling="no" frameborder="0" allowfullscreen="" /></iframe>
+   </div>
+  </div>
 
-  <h2>Get started</h2>
+  <h3>Get started</h3>
```

Comment:
Similarly, I don't think this margin is needed.
<img width="856" alt="Screen Shot 2022-11-16 at 11 57 41 AM" src="https://user-images.githubusercontent.com/5618806/202281625-41011d6a-ba9c-49d0-918d-f0562ac6320f.png">